wannierberri/system/system_w90.py

```diff
@@ -74,10 +80,15 @@ def __init__(
         self.num_wann = chk.num_wann
         self.wannier_centers_cart_auto = chk.wannier_centers
 
+        # Necessary ab initio matrices
         eig = EIG(seedname)
         if self.need_R_any(['AA','BB']):
             mmn = MMN(seedname, npar=npar)
-
+        if self.need_R_any(['CC']):
+            uhu = UHU(seedname)
+        if self.need_R_any(['CC','FF','FF_tot']):
```

Comment:
Do we need uiu, if we need only the `CC` ? I guess, it is not needed, e.g. in the case of only orbital moment